book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rents
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rents = []
+  end
+end
```

Comment:
- [ ] In your `book.rb`, kindly add a method that creates a new instance of the `Rental` class. Remember the relationship between `Book` and `Person` is a many-many relationship and they are joined through `Rentals`. This means that for a `Book` to connect with the `Person`, a `Book` should have many `Rentals` and a `Person` should have many `Rentals`. Kindly create this method that creates a new instance of Rental in the `Book` class.

#### _**Hint:**_ The method should have parameters for an instance of `Person`, and the `date`. Then when you create the instance of `Rental`, you can pass `self`. 👍

Reply:
Roger!